src/app/backend/handler/apihandler.go

```diff
@@ -584,6 +586,16 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 			To(apiHandler.handleLogFile).
 			Writes(logs.LogDetails{}))
 
+	// Plugin routes
```

Comment:
Would be nice to have these routers in a separate package. We moved, for example, some routes to the `integration` package.

Reply:
Moved to `plugin/handler.go`

Reply:
I don't see `plugin/handler.go` file.